### DISCUSSION TITLE
i want use windows form ， but can not load

### DISCUSSION THREAD

**Discussion by furacas** [BepInEx/BepInEx#432] [Help and Q&A] [answered]
i want use windows form ， but can not load ， i had put windows.form.dll in plugins path

System.IO.FileNotFoundException: Could not load file or assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.

**Comment by michieal** [comment, marked as answer]
Aside from my inherent curiosity as to why you want to use WinForms... I will point out that WinForms only works on Windows Devices, and only on the .NET versions up to 4.8-ish. Despite how it looks, Unity does not use any of that for it's compiler / builds. And, depending on the build settings, it's either using mono-bleeding edge, or it has disasembled all of the IL code into cpp, and then compiled that.

So, there's the main reason as to "why". Now for an actual answer to get past it. Why not use the Unity UI and make an user interface for the data that you wish to display, and add that into your "mod"?

**Reply by furacas** [reply]
Thank you for your answer

**Reply by michieal** [reply]
you're welcome :)
